client/src/Movies/SavedList.js

```diff
@@ -12,7 +13,14 @@ export default class SavedList extends Component {
         {this.props.list.map(movie => (
           <span className="saved-movie">{movie.title}</span>
         ))}
-        <div className="home-button">Home</div>
+          <NavLink exact to={"/"} history={"/"}> 
```

Comment:
if your just passing a string you dont need these curly braces, curly braces are only needed for values that need to be evaluated like an if statment, a function, or a stored variable.
```suggestion
          <NavLink exact to="/" history="/"> 
```

Reply:
oh yeah, I was using a different path, but then decided to rename it, and forgot to remove the curly braces, thanks